specification/components/r4/schemas/responses/searchServiceRequest/200Response.yaml

```diff
@@ -96,7 +95,19 @@ content:
                       - match
                     example: 'match'
     examples:
-      minimal-example:
-        summary: One UBRN associated with one ServiceRequest (referral)
+      minimal-example-empty:
+        summary: An empty bundle might be returned when the search is not successful
```

Comment:
Probably want to tweak this wording as this is still a successful response.

For example:
An empty bundle might be returned when the search completes but matches no requests.

Reply:
Agreed. I think my idea was to try to include the scenario of having no LR in this wording, hence 'search is not successful' but 

> An empty bundle might be returned when the search completes but matches no requests.

sounds better.